### ISSUE TITLE
ios can not complie

### ISSUE THREAD

**Issue by anfernees-iphone** [actorapp/actor-platform#46]
little info about the j2objc , always compile fail, more detail?

**Comment by prettynatty** [comment]
cc @ex3ndr

**Comment by prettynatty** [comment]
@anfernees-iphone, what error do you get?

**Comment by anfernees-iphone** [comment]
@prettynatty J2ObjC_header.h file not found.    could u  give me some hints? thanks

**Comment by ex3ndr** [comment]
You need to install j2objc to your machine and set "J2OBJC_HOME" environment variable.

**Comment by mad102190** [comment]
I'm getting the same issue. I installed j2objc and set the environment variable. Tried running pod install again and it failed because the dist file in the /Pods/j2objc-local folder was an invalid alias. I deleted it, recreated it pointing to the correct place, and then ran `pod install` again. This time the pods installed but the project still won't build with the error that @anfernees-iphone mentioned. "J2ObjC_header.h file not found"
